engine/apps/api/tests/conftest.py

```diff
@@ -56,11 +55,3 @@ def _make_alert_groups_all_statuses(alert_receive_channel, channel_filter, alert
         return resolved_alert_group, ack_alert_group, new_alert_group, silenced_alert_group
 
     return _make_alert_groups_all_statuses
-
-
-@pytest.fixture()
-def mock_alert_shooting_step_post_alert_group_to_slack(monkeypatch):
```

Comment:
Tests pass without this fixture

Reply:
It may be there to avoid making requests to slack 🤔 could you double-check it please?

Reply:
it's _possible_ that that is the case, but this fixture is only used in 2 spots ([repo code search](https://github.com/search?q=repo:grafana/oncall%20mock_alert_shooting_step_post_alert_group_to_slack&type=code)).

Both of [those tests](https://github.com/grafana/oncall/blob/4826291856f35b96207cefed494a7cd14a7feb95/engine/apps/api/tests/test_alert_receive_channel.py#L415-L459) are internal API tests which simply test a few cases on the `GET /alert_receive_channel` endpoint 🤔 

Maybe it used to be relevant but we forgot to remove it after a refactor? I stumbled across this when deciding to rename `AlertShootingStep`

Reply:
(in any event, if we were making calls to Slack somewhere along the line; they'd likely fail since our tests on CI are not configured w/ any Slack credentials)

Reply:
but you bring up a valid point, these tests shouldn't be making external network calls. We could _maybe_ do something like [this](https://stackoverflow.com/questions/18601828/python-block-network-connections-for-testing-purposes) to prevent that

Reply:
Thanks for checking! In this case agree yes, it can be deleted, I guess 👍 
I agree, it makes sense to prevent tests making external network calls just in case